digiroad2-oracle/src/main/resources/db/migration/V1_14__change_lane_work_list_link_id_to_varchar.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE lane_work_list ALTER COLUMN link_id TYPE VARCHAR(40);
```

Comment:
lisätäänkö tänne se vvh_id homma myös vai ylikirjoitetaanko tää link_id ? @anttiahopeltositowise @SWJPTulijoki

Reply:
linkId importer batch pitäisi varmaan lisätä tää taulu myös.

Reply:
En näe itse tarvetta säilyttää VVH id:tä

Reply:
vaatii erikois käsittelyn sitten tossa linkId importer batch

Reply:
sinäänsä ominaisuus uus joten ei varmaan menetetä mitään vaikka vvh id ylikirjoitetaan

Reply:
jälkeen päin tarkistus tietty vähän hankalampaa sitten